app/assets/javascripts/templates/data_portal/modals/modal-download-all.jst.ejs

```diff
@@ -0,0 +1,13 @@
+<div class="c-modal-download-all">
+  <h2 class="title -dark">Download</h2>
+  <p class="description">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Minima cumque asperiores nam enim! Veniam reprehenderit possimus, iure labore explicabo ipsum ducimus eligendi</p>
+
+  <form action="#" onSubmit class="c-form js-form">
+    <input type="email" name="email" class="js-email" placeholder="E-mail address" required />
+    <div class="terms-container">
+      <input type="checkbox" name="terms-and-conditions" class="js-terms" /><label for="terms-and-conditions">I accept to the <a href="/terms-of-use" target="_blank" rel="noopener noreferrer">terms and conditions.
```

Comment:
You can avoid having the path here. Here are the steps to get it from rails:
1. Change the route in `routes.rb`: `get 'terms-of-use', to: 'static_pages#terms_of_use', as: 'terms_of_use'`
2. Rename `modal-download-all.js` by `modal-download-all.js.erb`
3. Add `<% environment.context_class.instance_eval { include Rails.application.routes.url_helpers } %>` to the top of `modal-download-all.js.erb`
4. Pass the path from the JS: `this.options.content = this.contentTemplate({ termsLink: '<%= terms_of_use_path %>' });`
5. Update your template